### DISCUSSION TITLE
How can I use exclude-names-regex

### DISCUSSION THREAD

**Discussion by ighack** [apache/skywalking#9063] [Q&A]
How can I use exclude-names-regex

I get these error:
 Successful rate of endpoint /actuator/health in moutai-mip-flowcompute is lower than 80% in 2 minutes of last 10 minutes.
 Successful rate of endpoint /${server.error.path:${error.path:/error}} in moutai-mip-flowcompute is lower than 80% in 2 minutes of last 10 minutes.
 Successful rate of service instance 2f41afc8054341a5a8a4b4af3d202034@172.110.160.122 of moutai-mip-flowcompute is lower than 80% in 2 minutes of last 10 minutes.
 

I use 

exclude-names-regex:
  - moutai-mip-flowcompute
  
that is error

exclude-names-regex:
  - moutai-mip-flowcompute
  
that is error yet

someone can give me a sample.

```
#  endpoint_sla_rule:
#    metrics-name: endpoint_sla
#    exclude-names: 
#      - moutai-mip-flowcompute
#    op: "<"
#    threshold: 8000
#    period: 10
#    count: 2
#    silence-period: 3
#    message: Successful rate of endpoint {name} is lower than 80% in 2 minutes of last 10 minutes
#  service_instance_sla_rule:
#    metrics-name: service_instance_sla
#    exclude-names-regex: 
#      - /moutai-mip-flowcompute/
#    op: "<"
#    threshold: 8000
#    period: 10
#    count: 2
#    silence-period: 3
#    message: Successful rate of service instance {name} is lower than 80% in 2 minutes of last 10 minutes

```

 I want to exclude  server name is "moutai-mip-flowcompute"

**Comment by wu-sheng** [comment]
Regex is a common concept in computer science, how about learning on your first?

**Reply by ighack** [reply]
can you give me a sample
